### ISSUE TITLE
Totalausfall benachrichtigt nicht über Lösung

### ISSUE THREAD

**Issue by hansmorb** [wielebenwir/commonsbooking#1346]
Zum reproduzieren:
Artikel buchen -> Totalausfall erstellen : Buchender erhält zwei E-Mails, eine zum Totalausfall und eine zur Stornierung
JETZT
Totalausfall auf gelöst: Buchender erhält keine Benachrichtigung, dass das Problem mittlerweile gelöst ist.

Erwartetes Verhalten: Nutzender wird darüber benachrichtigt, dass das Problem gelöst ist. Sollte am besten so formuliert sein, dass der Nutzende über seine

**Comment by chriwen** [comment]
kann es sein, dass nach dem ändern des Status nochmal erneut der "Senden"-Button geklickt werden muss? Hattest du das so probiert?

**Comment by hansmorb** [comment]
Ja, habe ich probiert. Ich glaube das Problem liegt in Zeile 262 vom Restriction Model, dort werden zwar cancelled Bookings abgerufen, aber mit der Funktion \CommonsBooking\Repository\Booking::getByRestriction . Und die default Werte von der Funktion sucht nur nach Confirmed Buchungen

![image](https://github.com/wielebenwir/commonsbooking/assets/6433480/d3ba593f-cc89-4010-8b8c-dfdf13218c43)

Wäre es erwartetes Verhalten, dass dort auch alle cancelled Buchungen benachrichtigt werden?

**Comment by chriwen** [comment]
Achso. Das war so by Design gewünscht, dass die nicht benachrichtigt werden.  Wir haben gesagt, wenn Buchungen storniert sind, müssen sich die Personen anderweitig kümmern. Die Software braucht dann nicht nochmal informieren. Die Annahme hier hier ist dass diese Info nur für einen sehr geringen Teil interessant wäre. Alle anderen können entsprechend ihrer Pläne sich nach alternativen Rädern umschauen. Nur bei vorübergehenden Einschränkungen kann das evtl. Sinn machen.

**Comment by hansmorb** [comment]
Jup, deshalb habe ich das auch erstmal als Feature markiert. Wahrscheinlich bräuchte man dafür wenn dann ein separates Template. Aber das ist vielleicht was für die lange Bank. Wollte es nur mal dokumentiert haben weil ich da schon ein paar Mal drüber gestolpert bin.
